gdraw/gfilechooser.c

```diff
@@ -510,19 +459,6 @@ static void GFileChooserScanDir(GFileChooser *gfc,unichar_t *dir) {
     GGadgetSetList(&gfc->directories->g,ti,false);
     GGadgetSelectOneListItem(&gfc->directories->g,0);
 
-    if ( gfc->outstanding!=NULL ) {
```

Comment:
What practical purpose did the outstanding logic serve?

Reply:
It was to support the async nature of FTP, like you can't get a response back immediately to directory operations.

Since we removed it, there's no need for this any more.